rasa/core/agent.py

```diff
@@ -593,8 +593,8 @@ def toggle_memoization(self, activate: bool) -> None:
     def continue_training(
         self, trackers: List[DialogueStateTracker], **kwargs: Any
     ) -> None:
-        warnings.warn(
-            "Continue training will be removed in the next release. It won't be "
+        raise_warning(
+            "Continue training will be removed in the 2.0 release. It won't be "
```

Comment:
optional: maybe it makes sense to mark the function as deprecated? is it exposed to the users?

Reply:
mhm not sure what you mean, there isn't a `@Deprecated` like in java 😅